### ISSUE TITLE
Set up two OC problems sequentially

### ISSUE THREAD

**Issue by msparapa** [Rapid-Design-of-Systems-Laboratory/beluga#69]
Essentially continuation on the state eoms. I was thinking solving the planar hypersonic problem from the start, then feeding the final solution into a 3dof problem. Should make solving 3dof problems a lot faster. Plus this might be an important feature/test to have.

**Comment by SeanMatthewNolan** [comment]
Another related feature that I have wanted for a while is the ability to change the constraints sequentially in a simpler way. For example, solve a problem with free terminal velocity (which is easier to solve) than apply a terminal velocity constraint. Currently this require two input files and know how the solver handles constraints so you can add an appropriate parameter to the solution to the 1st solution.

**Comment by msparapa** [comment]
Added a new file at

```
examples/3dof/planarto3dof.py
```

It solves the same problem as `hypersonic3DOF.py` but uses the planar problem as a start, then feeds that into the full 3dof problem. It took 124 seconds to run as opposed to 885 seconds. I noticed some higher level issues, like #76, that will affect applications when we try to import `beluga` into other programs and potentially use it in a loop. That's a much bigger problem, though.
